lib/gitlab/client/jobs.rb

```diff
@@ -0,0 +1,158 @@
+class Gitlab::Client
+  # Defines methods related to projects.
+  # @see https://docs.gitlab.com/ee/api/jobs.html
+  module Jobs
+    # Gets a list of Jobs for a Project
+    #
+    # @example
+    #   Gitlab.jobs(1)
+    #   Gitlab.jobs("project")
+    #
+    # @param  [Integer, String] id The ID or name of a project.
+    # @return [Array<Gitlab::ObjectifiedHash>]
+    def jobs(project_id, scope='created')
+      get("/projects/#{url_encode project_id}/jobs", query: {scope: scope})
+    end
+
+    # Gets a list of Jobs from a pipeline
+    #
+    # @example
+    #   Gitlab.pipeline_jobs(1, 2)
+    #   Gitlab.pipeline_jobs("project", 2)
+    #
+    # @param  [Integer, String] The ID or name of a project.
+    # @param  [Integer]  the id of the pipeline
+    # @return [Array<Gitlab::ObjectifiedHash>]
+    def pipeline_jobs(project_id, pipeline_id)
```

Comment:
This can also take the scope, optionally, so do the same as you do for your `jobs` method.